CONTRIBUTING.md

```diff
@@ -341,7 +318,7 @@ Earning this privilege should be considered to be no small feat and is by no
 means guaranteed by any quantifiable metric. It is a symbol of great trust of
```

Comment:
```suggestion
means guaranteed by any quantifiable metric. Serving as a code owner is a symbol of great trust from 
```

Reply:
let's state the subject here, since the naked "it" allows ambiguity (especially for this awesome message)
I'm on a mission to remove all unclear references :), although "it" isn't clearly included in a developer documentation style guide, the same principle of "the naked it" applies to:

> this, that
> Where possible, put a noun after this or that for clarity. If doing so results in clunky prose, then don't do it; but even then, try thinking about what the noun would be. If you aren't sure what noun this or that (or in this case "It") refers to, then consider rephrasing.